DemoApp/Sources/Components/Stats/ParticipantStatsView.swift

```diff
@@ -0,0 +1,99 @@
+//
+// Copyright © 2023 Stream.io Inc. All rights reserved.
+//
+
+import StreamVideo
+import StreamVideoSwiftUI
+import SwiftUI
+
+struct ParticipantStatsView: View {
+
+    @StateObject var viewModel: ParticipantStatsViewModel
+    @Injected(\.colors) var colors
+        
+    var gridItem: GridItem {
+        GridItem.init(
+            .adaptive(minimum: itemSize + 2 * spacing),
+            spacing: spacing,
+            alignment: .topLeading
+        )
+    }
+    
+    init(call: Call, participant: CallParticipant) {
+        _viewModel = StateObject(
+            wrappedValue: ParticipantStatsViewModel(
+                call: call,
+                participant: participant
+            )
+        )
+    }
+    
+    var body: some View {
+        LazyVGrid(columns: [gridItem, gridItem], spacing: spacing) {
+            ForEach(viewModel.statsEntries) { entry in
+                VStack(spacing: spacing) {
+                    Text(entry.title)
+                        .font(.caption)
+                        .lineLimit(1)
+                    Text(entry.value)
+                        .font(valueFont)
+                }
+                .frame(width: itemSize, height: itemSize / 2)
+                .padding(.all, spacing)
+                .background(Color(colors.background1))
+                .cornerRadius(8)
+            }
+            Button {
+                viewModel.allStatsShown = true
+            } label: {
+                Text("All stats")
+                    .font(valueFont)
+                    .foregroundColor(colors.text)
+                    .frame(width: itemSize, height: itemSize / 2)
+                    .padding(.all, spacing)
+                    .background(Color(colors.background1))
+                    .cornerRadius(8)
+            }
+
+        }
+        .frame(width: 2 * itemSize + 4 * spacing)
+        .padding()
+        .background(colors.callControlsBackground)
+        .cornerRadius(8)
```

Comment:
```suggestion
.clipped()
.clipShape(RoundedRectangle(cornerRadius: 8))
```

I think it's better to use the clipShape to avoid artifacts getting rendered outside of the bound box

Reply:
done